server/routes/gcp/auth.py

```
@@ -224,6 +224,214 @@ def force_disconnect_gcp(user_id):
+                "error": f"Missing required field '{field}' in service account JSON"
+            }), 400
+
+    client_email = sa_info["client_email"]
+    home_project_id = sa_info["project_id"]
+
+    try:
+        creds = service_account.Credentials.from_service_account_info(
+            sa_info,
+            scopes=["https://www.googleapis.com/auth/cloud-platform"],
+        )
+    except Exception as exc:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)